LBH.AdultSocialCare.Api/V1/Boundary/Common/Request/FundedNursingCareUpdateRequest.cs

```diff
@@ -0,0 +1,43 @@
+using System;
```

Comment:
I think we need CarePackageID and UserID. In this way, everyone can update all reclaims but for now, let's put TODO item. We have to handle this with another UserStory which is implemented Role-based action

Reply:
When updating FNC carePackageId and UserId won't be changed so thats why I excluded those prop. Is that wrong ?

Reply:
That's right. UserId and carePackageId should be part of domain entity.